app/controllers/message_templates_controller.rb

```diff
@@ -4,15 +4,15 @@ class MessageTemplatesController < ApplicationController
   def recipient_selector
     authorize(@message_template, policy_class: MessageTemplatePolicy)
 
-    @recipients_list = @message_template.recipients
+    @recipients_list = @message_template.recipients.take(10)
```

Comment:
Toto moze byt potencialne nedeterministicke spravanie lebo tam nie je ziadny order. vid https://stackoverflow.com/questions/18496421/take-vs-first-performance-in-ruby-on-rails

order vsak moze spomalovat tu query avsak myslim, ze to nebude problem. dajme tam first